packages/zowe-explorer-api/src/profiles/ProfilesCache.ts

```diff
@@ -292,7 +311,7 @@ export class ProfilesCache {
         return newProfile.profile;
     }
 
-    protected async refreshConfig(apiRegister: ZoweExplorerApi.IApiRegisterClient): Promise<void> {
+    protected async refreshConfig(apiRegister?: ZoweExplorerApi.IApiRegisterClient): Promise<void> {
```

Comment:
If the `apiRegister` parameter is changed to be optional, we might want to add a check that it is defined before it is used at line 319 (`for (const type of apiRegister.registeredApiTypes()) {...}`). Would we use an alternative way to get the profile types if `apiRegister` is not passed in?